src/lib/mdns/minimal/ResponseSender.h

```diff
@@ -89,7 +89,10 @@ class ResponseSendingState
 class ResponseSender : public ResponderDelegate
 {
 public:
-    ResponseSender(ServerBase * server, QueryResponderBase * responder) : mServer(server), mResponder(responder) {}
+    static constexpr size_t kMaxQueryResponders = 3;
```

Comment:
Does this correspond to the max number of subtypes or the number of service types?

Reply:
Max number of service types - operational, commissionable node, commssioner. I've hardcoded it here - technically I suppose I could put it as a template, but that makes things a bit messier.

Reply:
Isn't there a distinct service advertised for each fabric.  So the number of advertisements is actually at least `n fabrics + 1 + 1`, no?

Reply:
yup - that's correct. Right now we don't have any multi-admin stuff, but you're right that we should be pro-active for when that rolls in for TE4. I think the real solution here is to template it it somehow, but I'm not sure the SDK in general has a concept of # supported fabrics at the moment. It'll have to eventually though.

Proposal for now - per https://github.com/CHIP-Specifications/connectedhomeip-spec/issues/2551, the number on the table right now for minimum # fabrics is 5. Suggest I update this to be hardcoded 7 with a comment linking back to the spec discussion. I opened an issue here to track updating this to get the real number from the SDK config once that work lands. Issue is here: https://github.com/project-chip/connectedhomeip/issues/8000. Does that work for you?

Reply:
Works for me.